bin/auto_test_snap.cmd

```diff
@@ -1,53 +1,11 @@
 @ECHO OFF 
 setlocal enabledelayedexpansion
 
-REM set important env vars
-IF DEFINED PFTT_SHELL GOTO :skip_set_env
-CALL %~dp0set_env.cmd
-:skip_set_env
+SET branch[0]=7.2
+SET branch[1]=7.3
+SET branch[2]=7.4
 
-if not exist %PHP_BUILDS% (
-	md %PHP_BUILDS%
-)
-
-SET branch[0]=7.1
-SET branch[1]=7.2
-SET branch[2]=7.3
-SET branch[3]=7.4
-SET cpu[0]=x64
-SET cpu[1]=x86
-SET cpu[2]=x64
-SET cpu[3]=x86
-SET thread[0]=NTS
-SET thread[1]=NTS
-SET thread[2]=TS
-SET thread[3]=TS
-SET vc[0]=vc14
-SET vc[1]=vc15
-SET vc[2]=vc15
-SET vc[3]=vc15
-
-REM Check php-7.1 for new snap builds
-for /L %%i in (0,1,3) do (
+for /L %%i in (0,1,2) do (
```

Comment:
To avoid such changes in the future, maybe we should apply:
````.patch
 bin/auto_test_snap.cmd | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/bin/auto_test_snap.cmd b/bin/auto_test_snap.cmd
index 005c607d3d..0c5235758e 100644
--- a/bin/auto_test_snap.cmd
+++ b/bin/auto_test_snap.cmd
@@ -1,11 +1,7 @@
 @ECHO OFF 
-setlocal enabledelayedexpansion
 
-SET branch[0]=7.2
-SET branch[1]=7.3
-SET branch[2]=7.4
+SET branches=7.2 7.3 7.4
 
-for /L %%i in (0,1,2) do (
-	SET branch=!branch[%%i]!
-	call %~dp0test_snap.cmd !branch!
+for %%i in (%branches%) do (
+	call %~dp0test_snap.cmd %%i
 )
\ No newline at end of file
````

Reply:
Committed change and pushing.